Cargo.toml

```diff
@@ -6,5 +6,24 @@ members = [
     "ic-utils",
     "ic-certification",
     "icx",
-    "ref-tests"
+    "ref-tests",
 ]
+
+[workspace.package]
+version = "0.23.1"
+authors = ["DFINITY Stiftung <sdk@dfinity.org>"]
+edition = "2021"
+repository = "https://github.com/dfinity/agent-rs"
+rust-version = "1.60.0"
```

Comment:
This should be 1.64.0 at minimum (probably 1.65.0 to correspond to rust-toolchain.toml); older versions of Rust won't be able to parse these crates.